### DISCUSSION TITLE
Discussion title does not markdown render

### DISCUSSION THREAD

**Discussion by avelino** [community/community#4272] [Discussions]
When you put markdown markup in the title of an issue the markdown is rendered, when you put markdown markup in the title of discussions it is not

**Issue:** "\`SET ROLE`"

![Screen Shot 2021-06-10 at 10.11.24\|690x83](https://aws1.discourse-cdn.com/github/original/3X/2/c/2c30d35bc1e891f1fb58d91a96634f606ac8de4a.png)
https://github.com/prest/prest/issues/548

**Discussion:** "\`SET ROLE`"

![Screen Shot 2021-06-10 at 10.11.17\|690x88](https://aws1.discourse-cdn.com/github/original/3X/5/6/5649b1e3a590c5996fc725a106fb36f291f51031.png)
https://github.com/prest/prest/discussions/549

_ref: https://github.community/t/discussion-title-does-not-markdown-render/184444_

**Comment by pubmikeb** [comment]
I've also paid attention, that Discussion titles don't support Markdown text formatting.
In sake of consistency, it would be great to enable Markdown for Discussions as well.
